### ISSUE TITLE
Can't find variable: AudioContext

### ISSUE THREAD

**Issue by felix-albrecht** [ondras/oscope#4]
I was trying to get this working, but ran into the following error:

`Line 7:32 Can't find variable: AudioContext`

This is for the online test as well as running it locally.
I have Safari 14 on macOS 10.14.

EDIT: On Google Chrome, it works.

**Comment by felix-albrecht** [comment]
So apparently, Safari doesn't support AudioContext. Bummer!
But, awesome project!

**Comment by ondras** [comment]
Safari seems to support a [prefixed version](https://caniuse.com/audio-api). I do not have access to Safari so I cannot really test that. @gopromaster, can you please confirm or reject this hypothesis?

**Comment by felix-albrecht** [comment]
Yes, it works partially. I replaced `AudioContext()` with `webkitAudioContext()` in app.js.

Thanks for the suggestion.

EDIT: Not everything works: Microphone Input and Sound files (local and remote) don't respond.
Web Audio Signal doesn't show a moving graph, and the sound doesn't change on slider moves.
Math function works great, though.
